projects/ngx-launcher/src/lib/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.ts

```diff
@@ -86,10 +77,15 @@ export class ProjectProgressCreateappNextstepComponent implements OnChanges, OnD
   retry() {
     const failedStep = this.lastCompleted;
     this.projectSummaryService.setup(
-      this.launcherComponent.summary, failedStep).subscribe(result => {
-        this._progress = null;
+      this.launcherComponent.summary, failedStep).subscribe(val => {
         this.errorMessage = null;
-        this.ngOnChanges({'statusLink': new SimpleChange('', result.uuid_link, false)});
+        if (!val || !val['uuid_link']) {
+          this.errorMessage = 'Invalid response from server!';
+        }
+        for (const step of this._progress.filter((s) => !s.completed)) {
+          step.error = false;
+        }
+        this.launcherComponent.statusLink = val['uuid_link'];
```

Comment:
will the `ngOnChanges` be called when you only do this?

Reply:
@edewit it does, you can try in the demo